packages/core/src/server/generateGraphqlSchemaFile.ts

```diff
@@ -0,0 +1,22 @@
+import { GraphQLSchema } from 'graphql'
+import { writeFileSync } from 'fs-extra'
+import { printSchemaWithDirectives } from '@graphql-tools/utils'
+
+import { apiSchema } from './schema'
+
+async function createGraphqlSchemaFile(apiSchema: Promise<GraphQLSchema>) {
```

Comment:
This file seems to be in the incorrect place. I don't think we need to do anything now but we might want to keep an eye out for future "tasks".

Reply:
I agree, I left it there for the lack of a better place.